pkg/adapters/grpc/event_grpc_test.go

```diff
@@ -17,24 +17,25 @@ package grpc_test
 import (
 	"context"
 	"fmt"
+
 	"github.com/nitric-dev/membrane/pkg/adapters/grpc"
 
 	v1 "github.com/nitric-dev/membrane/interfaces/nitric/v1"
-	"github.com/nitric-dev/membrane/pkg/sdk"
+	"github.com/nitric-dev/membrane/pkg/plugins/eventing"
```

Comment:
If we are doing this change, I think it would be a good time to consider changing this package name to events or event:

```github.com/nitric-dev/membrane/pkg/plugins/events```

Reply:
agreed, will pop that in as well

Reply:
done